Makefile

```diff
@@ -150,4 +151,13 @@ compile-requirements: .docker-build-pull
 check-requirements: .docker-build-pull
 	${DC} run --rm test pip list -o
 
-.PHONY: all clean build pull docs livedocs build-docs lint run stop kill run-shell shell test test-image rebuild build-ci test-ci fresh-data djshell run-prod build-prod test-cdn compile-requirements check-requirements
+######################################################
+# For use in local-machine development (not in Docker)
+######################################################
+
+install-local-python-deps:
+	pip install -U -r requirements/prod.txt --no-cache-dir
```

Comment:
Because the files specify all versions I don't think the `-U` does anything. Also why the `--no-cache-dir`? pip cache for our machines is a good thing. We only use that in our docker images to keep the image size down.

Reply:
Done in https://github.com/mozilla/bedrock/pull/11269/commits/c0348f6f6b2cbd5b62d5d8857dd4119266a9fc71